### PR TITLE
Build Script for jquery_ui-v1.4.0

### PR DESCRIPTION
**Pull request by HarithaPatchari054**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: d/drupal-jquery_ui/drupal-jquery_ui_ubi_8.5.sh [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+ 
+#Check if package exists
+if [ -d "$PACKAGE_NAME" ] ; then
+  rm -rf $PACKAGE_NAME
+  echo "$PACKAGE_NAME  | $PACKAGE_VERSION | $OS_NAME | GitHub | Removed existing package if any"   
+fi
+
+ if ! git clone $PACKAGE_URL $PACKAGE_NAME; then
+    	echo "------------------$PACKAGE_NAME:clone_fails---------------------------------------"
+		echo "$PACKAGE_URL $PACKAGE_NAME"
```
Comment (rajusah15) [inline d/drupal-jquery_ui/drupal-jquery_ui_ubi_8.5.sh:60]: make indentation uniform all pace.

File: d/drupal-jquery_ui/drupal-jquery_ui_ubi_8.5.sh [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+if ! composer install --no-interaction; then
+     	echo "------------------$PACKAGE_NAME:install_fails-------------------------------------"
+	echo "$PACKAGE_URL $PACKAGE_NAME"
+	echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Install_Fails"
+	exit 1
+fi
+
+#No test case available
+
+
```
Comment (rajusah15) [inline d/drupal-jquery_ui/drupal-jquery_ui_ubi_8.5.sh:78]:
```suggestion

```
remove space

File: d/drupal-jquery_ui/drupal-jquery_ui_ubi_8.5.sh [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=jquery_ui
+CORE_PACKAGE_NAME=drupal
+PACKAGE_URL=https://git.drupalcode.org/project/jquery_ui
+CORE_PACKAGE_URL=https://github.com/drupal/drupal
+#PACKAGE_VERSION is configurable can be passed as an argument.
+PACKAGE_VERSION=${1:-8.x-1.4}
+
+
```
Comment (rajusah15) [inline d/drupal-jquery_ui/drupal-jquery_ui_ubi_8.5.sh:29]:
```suggestion

```
keep uniform all places , remove space.

File: d/drupal-jquery_ui/drupal-jquery_ui_ubi_8.5.sh [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+        echo "------------------$PACKAGE_NAME:clone_fails---------------------------------------"
+        echo "$PACKAGE_URL $PACKAGE_NAME"
+        echo "$PACKAGE_NAME  |  $PACKAGE_URL |  $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Clone_Fails"
+        exit 1
+fi
+cd $PACKAGE_NAME
+git checkout $PACKAGE_VERSION
+cd ../../
+cd core/
+
```
Comment (rajusah15) [inline d/drupal-jquery_ui/drupal-jquery_ui_ubi_8.5.sh:75]: @HarithaPatchari054  go to path of package and install/build using composer  e.g: 'composer install' .

**Review by rajusah15 — COMMENTED** [review]
build the package prior to test.

File: d/drupal-jquery_ui/drupal-jquery_ui_ubi_8.5.sh (hunk 4 above)
Comment (HarithaPatchari054) [reply d/drupal-jquery_ui/drupal-jquery_ui_ubi_8.5.sh:75]: @rajusah15 This package doesn't contain composer.json

**Comment by prashantkhoje** [comment]
LGTM

**Pull request merged**